README.md

```diff
@@ -48,6 +48,30 @@ The following metadata formats are supported:
   - DDI 2.5 using metadataprefix ``oai_ddi25``.
   - OpenAIRE Datacite using metadataprefix ``oai_datacite``.
 
+The application exposes a /metrics endpoint, which provides certain
+statistics about the running instance of the application. This
+endpoint is provided by
+[prometheus-client](https://github.com/prometheus/client_python). Following
+metrics are exposed:
+
+  - Total number of external catalogue requests received counter via
+    `requests_total`.
+  - Number of external catalogue requests received per user-agent
+    counter `requests_per_user_agent_total`.
+  - Number of successful catalogue requests counter via
+    `requests_succeeded_total`.
+  - Number of failed catalogue requests counter via
+    `requests_failed_total`.
+  - Response time in milliseconds summary via `requests_duration`.
+  - Total number of records included gauge via `records_total`.
+  - Total number of records included without logically deleted records
+    gauge via `records_total_without_deleted`.
+  - Total number of distinct publishers gauge via `publishers_total`.
+  - Number of records included per Publisher gauge via
+    `publishers_counts`.
+  - Number of records included per Publisher without logically deleted
+    records gauge via `publishers_counts_without_deleted`.
```

Comment:
Formatting the metrics as a table would be much easier to read. It also makes the types easier to identify.

```suggestion
| Metric                              | Type    | Explanation                                                                                 |
| ----------------------------------- | ------- | ------------------------------------------------------------------------------------------- |
| `requests_total`                    | Counter | Total number of requests received                                                           |
| `requests_per_user_agent_total`     | Counter | Number of requests received per user-agent                                                  |
| `requests_succeeded_total`          | Counter | Number of successful requests                                                               |
| `requests_failed_total`             | Counter | Number of failed requests                                                                   |
| `requests_duration`                 | Summary | Response time in milliseconds                                                               |
| `records_total`                     | Gauge   | Total number of OAI-PMH records (includes records marked as deleted)                        |
| `records_total_without_deleted`     | Gauge   | Total number of OAI-PMH records (excludes records marked as deleted)                        |
| `publishers_total`                  | Gauge   | Total number of distinct publishers (defined by the repository's declared OAI-PMH base URL) |
| `publishers_counts`                 | Gauge   | Number of OAI-PMH records per publisher (includes records marked as deleted)                |
| `publishers_counts_without_deleted` | Gauge   | Number of OAI-PMH records per publisher (excludes records marked as deleted)                |
```